fastlane/Fastfile

```
@@ -0,0 +1,137 @@
+
+  desc "Test Build"
+  lane :build_test do |options|
+      gradle(
+            task: "clean assemble#{options[:variant]}Debug"
+          )
+  end
+
+  desc "Build and Distribute Debug APK to Firebase"
+  lane :build_and_distribute_debug do |options|
+
+   latest_release = firebase_app_distribution_get_latest_release(
```

RuboCop (1.69.1)

[convention] Extra empty line detected at block body beginning.

(Layout/EmptyLinesAroundBlockBody)